<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>930</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>854</width>
    <height>930</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>854</width>
    <height>930</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>854</width>
     <height>930</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>854</width>
     <height>930</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainLO">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="headerLO">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="leftHeaderLO">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="4" column="2">
           <widget class="QPushButton" name="currentPB">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);</string>
            </property>
            <property name="text">
             <string>RealTime</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(255, 255, 255);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>Num. Station</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="playPB">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);</string>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLCDNumber" name="dataTimeLCD">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(186, 189, 182);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>17</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(255, 255, 255);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>Data Time (UTC)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="stopPB">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);</string>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLCDNumber" name="numStaLCD">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(186, 189, 182);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
            <property name="digitCount">
             <number>6</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="rightHeaderLO">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="dateLB">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(255, 255, 255);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>DATE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDial" name="secDial">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
</string>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="notchTarget">
             <double>11.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="secLB">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(255, 255, 255);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>SECOND</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDial" name="hourDial">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
</string>
            </property>
            <property name="maximum">
             <number>23</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="notchTarget">
             <double>6.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDial" name="minDial">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
</string>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="notchTarget">
             <double>11.000000000000000</double>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QComboBox" name="eventListCB">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);</string>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Select a Available EEW Event</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="monChanCB">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>22</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);</string>
            </property>
            <property name="currentIndex">
             <number>3</number>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>East/West</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>North/South</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Up/Down</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Horizontal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Total(3D)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="hourLB">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(255, 255, 255);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>HOUR</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="minLB">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>98</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(46, 52, 54);
color:rgb(255, 255, 255);
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string>MINUTE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDial" name="dateDial">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>67</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
</string>
            </property>
            <property name="maximum">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="mapFR">
        <property name="minimumSize">
         <size>
          <width>814</width>
          <height>768</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>814</width>
          <height>768</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="mapLO">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
